.github/workflows/genericx86-64-ext.yml

```diff
@@ -53,11 +53,12 @@ jobs:
     if: (github.event.pull_request.head.repo.full_name == github.repository) == (github.event_name == 'pull_request')
     secrets: inherit
     with:
-      machine: genericx86-64-ext
+      machine: generic-amd64 # USE BALENA GENERIC TO TEST THIS PR
       deploy-environment: balena-staging.com
       # device-repo and device-repo-ref inputs should not be provided on device repos
-      device-repo: balena-os/balena-intel
+      device-repo: balena-os/balena-generic # USE BALENA GENERIC TO TEST THIS PR
       device-repo-ref: master
+      deploy-ami: true # FORCE AMI DEPLOYS ON PR TO TEST THIS PR
```

Comment:
We haven't actually enabled image signing in the yocto build here, so whatever we are testing with the AMI isn't secure boot (and might explain why it's working).